newIDE/app/src/Utils/ZoomUtils.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+const stepZoomFactor = Math.pow(2, 1 / 8);
+export const zoomInFactor = stepZoomFactor;
+export const zoomOutFactor = Math.pow(stepZoomFactor, -1);
+const continuousZoomFactor = Math.sqrt(stepZoomFactor);
+
+// TODO: Use absolute value of signal that should represent either:
+// - Mouse sensitivity
+// - MacOS scroll acceleration
+// Signal is usually WheelEvent.deltaY
+export const getContinuousZoomFactor = (signal: number): number =>
+  Math.pow(continuousZoomFactor, Math.sign(signal));
```

Comment:
Does it feel right without the `Math.sign`?

Reply:
Nope, it's too quick, and the macos scroll acceleration makes it hard to normalize the value between, let's say, 0.5 and 3. Moreover I don't know what are the values for deltaY usually.